src/common/MerjoonTransformer.ts

```diff
@@ -4,11 +4,11 @@ import crypto from 'node:crypto';
 export class MerjoonTransformer implements IMerjoonTransformer {
   static separator = '->'
   static parseTypedKey(key: string) {
-    const regex = /(UUID|STRING)\("([a-z0-9-_.\->[\]]+)"\)/;
+    const regex = /(UUID|STRING|Timestamp)\("([a-z0-9-_.\->[\]]+)"\)/;
```

Comment:
Actually let's change the requirements and make `Timestamp` to `TIMESTAMP` to keep things similar